orchagent/intfsorch.cpp

```diff
@@ -276,7 +351,8 @@ void IntfsOrch::doTask(Consumer &consumer)
 
         const vector<FieldValueTuple>& data = kfvFieldsValues(t);
         string vrf_name = "", vnet_name = "";
-
+        uint32_t mtu = 0;
+        bool adminUp = true;
```

Comment:
It is not recommended to have this default in orch. I would suggest to pass this from Intfmgr.

Reply:
Even if we do not explicitly set default value to adminUp, the variable has a value by default. I am thinking of the misconfiguration scenario that the CONFIG_DB VLAN_SUB_INTERFACE table does not have "admin_status" field. In that case, if it happens, the default value takes effect.

Looking into to code, Intfmgr transparently passes the field value tuples from CONFIG_DB VLAN_SUB_INTERFACE table to APPL_DB INTF_TABLE. So if "admin_status" field, for example, is missing in the CONFIG_DB VLAN_SUB_INTERFACE table, intfmgr does not add it at its handling.

Reply:
Added both, admin_status and mtu, to field value tuples set to APPL_DB, if they are missing in CONFIG_DB VLAN_SUB_INTERFACE table